jangaroo-maven/jangaroo-maven-plugin/src/main/java/net/jangaroo/jooc/mvnplugin/AbstractCompilerMojo.java

```diff
@@ -385,8 +400,9 @@ protected JoocConfiguration createJoocConfiguration(Log log) throws MojoExecutio
     configuration.setOutputDirectory(getClassesOutputDirectory());
     configuration.setLocalizedOutputDirectory(getLocalizedOutputDirectory());
     configuration.setApiOutputDirectory(getApiOutputDirectory());
-    configuration.setSourcesAreTests(isTestRun());
-    configuration.setFindUnusedDependencies(findUnusedDependencies());
+    configuration.setFindUnusedDependencies(findUnusedDependencies(staleMillis));
+    //todo: use proper directory
+    configuration.setDependencyReportOutputFile("target/dependencyReports/dependencyWarnings");
```

Comment:
What about some file extension like `.txt` or `.log`? Could make it easier to look into the file if someone's curious.

Reply:
Coming from linux i didnt really think about any file extensions. I can add that